components/statuslight/index.css

```diff
@@ -49,6 +49,10 @@ governing permissions and limitations under the License.
 
     /* margin on the dog is the opposite of the padding in order to align the dot as if the padding hack was not present */
     margin: var(--spectrum-statuslight-padding-bottom) var(--spectrum-statuslight-text-gap) var(--spectrum-statuslight-padding-top) var(--spectrum-statuslight-text-gap);
+
+    /* support high contrast/forced-color-adjust mode for dot */
+    -ms-high-contrast-adjust: none;
+    forced-color-adjust: none;
```

Comment:
This [CSS Color Module Level 4 property](https://www.w3.org/TR/css-color-adjust-1/#forced-color-adjust-prop) forces the `.spectrum-StatusLight::before` dot to render the `background-color` in high contrast mode and will work in Chromium Edge.